tools/check.py

```diff
@@ -200,7 +207,99 @@ def _validate_section_heading_order(self, ast_node=None, headings=None):
                 "the order specified by the template".format(self.filename))
 
         return (len(missing_headings) == 0) and \
-               valid_order and no_extra and correct_level
+            valid_order and no_extra and correct_level
+
+    def _validate_one_callout(self, callout_node):
+        """
+        Logic to validate a single callout (blockquoted section). Checks that:
+        * First child of callout box should be a header with known & CSS style
+        * Callout box must have at least one child after the heading
+        * Checks # times callout appears in document, min <= n <= max
+        """
+        heading_node = callout_node.children[0]
+        has_heading = self.ast.is_heading(heading_node, heading_level=2)
+
+        if not has_heading:
+            # TODO: Is it correct that all blockquote sections should have headings?
```

Comment:
Thanks as always for the helpful feedback.

I've updated the pull request to reflect the new definition of callouts: "blockquoted sections whose first child is a heading". 

Any blockquoted section that does not begin with a heading will be ignored by the validator.